dotnet-install.ps1

```diff
@@ -0,0 +1,346 @@
+#
```

Comment:
ah, never mind, I accidentally checked in one of the ones that got downloaded, will delete